examples/coolest-todos/apps/todo-shell/next.config.js

```diff
@@ -1,13 +1,15 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { HostWebpackPlugin } = require("@leanjs/webpack");
-const withTM = require("next-transpile-modules")([
-  "@leanjs/core",
-  "@leanjs/react",
```

Comment:
@luciopalmieri do we need this?

Reply:
I'm not sure, I think that from the point of view of the example the https://github.com/leanjs are third-party packages while https://github.com/my-org are their own packages

Reply:
You're right, Next needs to build `@my-org` packages because they are local packages in the repo and we are not publishing a build to a registry. `@leanjs` packages are like external packages because the `examples` folder is not part of the monorepo workspaces.

Reply:
ok